components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -677,6 +677,9 @@
 .error_updating_consents {
   margin-top: 1em;
   margin-left: 2em;
+}
+.view-title-required-not-granted {
+  font-size: 1.15em;
```

Comment:
[The whole 'before-patient-data' block (containing consents, permissions, studies) has `font-size: .87em`.](https://github.com/phenotips/phenotips/blob/master/components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml#L3791) I'm not sure it's needed or even helps in any way. Should we maybe remove that instead of adding a new rule?